README.md

```diff
@@ -0,0 +1,91 @@
+
+This will trigger a call to `DomainHost.withdraw(address to, uint256 amount)`.
+
+### `DomainHost.lift(uint256 wad)`
+
+Specify a new global debt ceiling for the remote domain. If the amount is an increase this will pre-mint ERC20 and stick it in the domain escrow, so the future minted DAI on the remote domain is backed. It will not remove escrow DAI upon lowering as that can potentially lead to unbacked DAI. Releasing pre-minted DAI from the escrow needs to be handled by the remote domain.
+
+This will back the pre-minted DAI by `vat.gems` that represent shares to the remote domain.
+
+This will trigger a call to `DomainGuest.lift(uint256 _lid, uint256 rad)` with `rad == wad * RAY`.
+
+### `DomainGuest.surplus()`
+
+Permissionless function which will push a surplus to the host domain if the delta is greater than the dust limit. It will exit the DAI, send it across the token bridge leaving it ready for being realized by governance.
+
+This will trigger a call to `DomainHost.surplus(uint256 _lid, uint256 wad)` with `wad >= dust`.
+
+### `DomainGuest.deficit()`
+
+Permissionless function which will push a deficit to the host domain if the delta is greater than the dust limit. It will send a message to the host's domain informing it that the guest is insolvent and needs more DAI. Due to the fact the guest could be compromised we need the Host to authorize the recapitalization operation with a call to `DomainHost.rectify()`.
+
+This will trigger a call to `DomainHost.deficit(uint256 _lid, uint256 wad)` with `wad >= dust`.
+
+### `DomainHost.accrue(uint256 _grain)`
+
+Authed function which will effectively move the accounted surplus to the buffer. It will generate new pre minted DAI if necessary to cover remote's debt (it is up to governance to pass the correct value).
+
+### `DomainHost.rectify()`
+
+Suck some DAI from the surplus buffer and send it to the Guest to cover the bad debt.
+
+This will trigger a call to `DomainGuest.rectify(uint256 _lid, uint256 wad)`.
+
+### `DomainHost.cage()`
+
+Trigger shutdown of the remote domain. This will initiate an `end.cage()`. If you want to gracefully shutdown over a longer period you should call `DomainHost.lift(0)` to prevent new minting.
+
+This will trigger a call to `DomainGuest.cage(uint256 _lid)`.
+
+### `DomainGuest.tell()`
+
+Grabs the final debt level of this `dss` instance during global settlement. The reported value is the `cure` to report back to the host domain.
+
+This will trigger a call to `DomainHost.tell(uint256 _lid, uint256 value)`.
+
+### `DomainHost.exit(address usr, uint256 wad)`
+
+Used during global settlement to provide DAI holders with a share claim on the remote collateral. Mints a claim token on the remote domain which can be used in the remote `end` to get access to the collateral.
+
+This will trigger a call to `DomainGuest.exit(address usr, uint256 claim)`. Where `claim` equals to `wad * debt / grain`.
```

Comment:
A note here that we removed grain from DomainGuest so I want to make sure that this makes sense in context of that

Edit: I think grain being removed may leave this calculation lacking context? @gbalabasquer

Reply:
The calculation happens on L1, where `grain` still exists. Basically this math was previously done in L2 but when removing `grain` there was moved to L1.
Maybe there is something of the wording that could be better described.